test/system.spec.js

```diff
@@ -211,11 +211,36 @@ describe('System', function () {
           .then(supposedToFail)
           .catch(function (e) {
             expect(e)
-              .to.be.equal('dep error\n\tError evaluating ' + baseURL + 'test/loads/deperror.js');
+              .to.be.equal('Error evaluating ' + baseURL + 'test/loads/deperror.js\n\tdep error');
```

Comment:
As in previous comment, this should be reversed. We start with the error, moving out through the call site reporting.